### ISSUE TITLE
Enable IPv6 for the host network

### ISSUE THREAD

**Issue by 24mu13** [007revad/Synology_ContainerManager_IPv6#6]
Could this script potentially work also for **host** network, other than bridge one?

Even by enabling IPv6 on DSM, this is not supported (yet?) on Container Manager.

See: https://kb.synology.com/en-uk/DSM/tutorial/What_applications_on_Synology_NAS_are_IPv6_supported

**Comment by 007revad** [comment]
I was unable to get IPv6 working on the host network in Container Manager.

**Comment by 016** [comment]
@007revad quick update, in DSM7.2.1 for default the IPv6 is actived for host network in Container Manager, in UI it's "disabled", BUT if add any container to host network, it's working good, here is my test code.
`
//1.create docker container
docker run -d --restart=always --name ee_ddns_go \
 --network=host \
 -v /volume1/pathTo:/root\
 jeessy/ddns-go:v6.7.7

//2. testing in side docker container
docker exec -it ee_ddns_go /bin/sh
/app # ping6 v6.yinghualuo.cn
PING v6.yinghualuo.cn (2408:8763:0:983::8006): 56 data bytes
64 bytes from 2408:8763:0:983::8006: seq=0 ttl=52 time=81.086 ms
64 bytes from 2408:8763:0:983::8006: seq=1 ttl=52 time=85.232 ms
64 bytes from 2408:8763:0:983::8006: seq=2 ttl=52 time=84.898 ms
64 bytes from 2408:8763:0:983::8006: seq=3 ttl=52 time=81.757 ms

3. access by browser is also good
http://[240e:3a3:6e13:dpit:::92c4]:9876
`

thx for your great work, I will test your IPv6 script soon for open IPv6 on bridge network.

**Comment by 016** [comment]
@007revad hi, back to update, the script "[Synology_ContainerManager_IPv6]" is working good for access to docker container by IPv6 address, but it's not working when send request from the docker container to internet by IPv6, just like 
`working:
access by browser is also good
http://[240e:3a3:6e13:dpit:::92c4]:9876

not working:
testing in side docker container
docker exec -it ee_ddns_go /bin/sh
/app # ping6 v6.yinghualuo.cn
`

the only way make the ping6 working in docker container is use host network as I know right now. maybe macvlan will work too, I will test later.

**Comment by 016** [comment]
more update. for DSM7.2.2 don't need run script, for default the access by IPv6 is active, just can't access internet by ipv6 in container. @007revad

**Comment by 007revad** [comment]
@016 
Did manage to get internet access from a container using ipv6?

**Comment by 016** [comment]
@007revad  nope, by default can only access by IPv6 from wan to docker container, if want to send a IPv6 request from docker container to wan, I found two options, A. use host, B. macvlan.
